docs/DOCKER.md

```diff
@@ -0,0 +1,52 @@
+## **Docker**
+
+- With the exeption of the database root password every parameter / environment variable is retrieved using .env
+- Change .env and docker-compose to specific values
+#### **Build image yourself**
+- Check your current user id using `id -u` , should be the owner of the files of this repo
+- Set `WWW_UID` argument in docker-compose.yml to that specific value.
+- Execute `docker-compose up --build --force-recreate --detach` to build and serve the app
```

Comment:
"Build image yourself"
Maar dit commando is voor de docker compose file runnen.
Ja hij bouwt de images maar he?